lab3/src/main/java/agh/ics/oop/Animal.java

```diff
@@ -0,0 +1,40 @@
+package agh.ics.oop;
+
+class Animal {
+    private MapDirection orientation = MapDirection.NORTH;
+    private Vector2d position = new Vector2d(2, 2);
+    private final Vector2d rightTopCorner = new Vector2d(4, 4);
```

Comment:
fajnie że to Pan wyciągnął, tego typu elementy to stałe, warto oznaczać je jako `static final` i pisać nazwy DUŻYMI_LITERAMI. Będę o tym jeszcze mówił na zajęciach.